src/style.css

```diff
@@ -1,3 +1,83 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  width: 100%;
+  background-color: white;
+  position: relative;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: right;
+  align-items: center;
+  height: 60px;
+  padding: 0;
+  margin-right: 50px;
+}
+
+.moyo-logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+  padding: 0;
+  margin: 0;
+
+}
+
+.nav-link {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+  height: 15px;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+}
+
+.nav-link:hover {
+  color: #00acdc;
+}
+
+.nav-item-list {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+}
+
+.is-active {
+  position: relative;
+}
+
+.is-active::after {
+  position: absolute;
+  background-color: #00acdc;
```

Comment:
better to move all repeatable colors to vars and reuse them everywhere

Reply:
I dont know how to to that.

Reply:
I guess it was in one of the previous topics (maybe in HTML+CSS basic): https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

FYI :slightly_smiling_face: